sections/futures/Trade/ManagePosition.tsx

```diff
@@ -99,7 +100,7 @@ const ManagePosition: React.FC<ManagePositionProps> = ({
 			</div>
 
 			{(orderError || error || previewTrade?.showStatus) && (
-				<ErrorMessage>{orderError || error || previewTrade?.statusMessage}</ErrorMessage>
+				<Error>{orderError || error || previewTrade?.statusMessage}</Error>
```

Comment:
Are `orderError` and `error` user friendly messages or should we format these?

Reply:
These errors are mapped elsewhere from the codes in the contract to "clean" messages.